01-Login/exec.ps1

```diff
@@ -0,0 +1,2 @@
+docker build -t auth0-samples/spring-mvc-00login:latest .
+docker run -p 3000:3000 -v ~/.m2:/home/gradle/.gradle -it auth0-samples/spring-mvc-00login:latest
```

Comment:
this -v option is mounting the local directory `~/.m2` into the container folder `/home/gradle/.gradle`. Hosts should not need to install anything other than Docker.

Reply:
The gradle image is defining a volume in `/home/gradle/.gradle` directory, if we don't mount the volume dependencies installation won't be cached.

Reply:
There's no need to cached them. But if you think the opposite, adding a `RUN gradle clean build` command to the docker file would make them already available in the image on the run time, right?